package.json

```diff
@@ -6,58 +6,31 @@
 		"checkFormat": "npm run prettier -- --list-different",
 		"coveralls": "node node_modules/coveralls/bin/coveralls.js < coverage/lcov.info",
 		"format": "npm run prettier -- --write",
-		"jest": "cross-env NODE_ENV=test jest",
 		"lerna": "lerna bootstrap -- --no-optional --no-package-lock",
 		"link": "lerna run link",
 		"lint": "eslint packages/**/*.js examples/**/*.js scripts/**/*.js",
 		"pa11y": "pa11y-ci ./packages/clay-*/demos/a11y.html",
 		"prettier": "prettier-eslint packages/**/**/*.js examples/**/**/*.js scripts/**/**/*.js",
-		"start": "http-server . -p 4000",
-		"test": "npm run build && npm run jest && npm run pa11y",
+		"test": "lerna run test && npm run pa11y",
```

Comment:
Just putting my thoughts about it:

- One of the points I like about having the jest running at the root is that we will only have one jest startup running and we can reap the coverage much better than terms in each package its coverage... This facilitates the information on https://coveralls.io/github/liferay/clay.
- We will have many components here still, so running the test for each project will take a long time for you to know if you have not broken another component that depends on it.
- When running the tests in this way, we will have the high boot time in each package, since we raise the jest all the time to test the package, maybe increase the CI time.
- I like to run the tests at the root because I can select which tests I want to run when I'm working on just one project. `yarn jest packages/clay-badge` for example.

Reply:
Ah yeah that makes sense. I think if we hoist devDependencies like Greg had suggested then we can also probably run a single instance of Jest at both the root and package levels. Ill look into this.